playwright/tests/pageTypes/pageTypeBuilderFields.spec.ts

```diff
@@ -491,24 +491,31 @@ test("I can see and copy the code snippets for nested groups and their sub field
   );
 });
 
-test("I cannot delete default UID field for reusable page type", async ({
+test("I cannot see default UID field for single page type", async ({
   pageTypesBuilderPage,
-  reusablePageType,
+  singlePageType,
 }) => {
-  await pageTypesBuilderPage.goto(reusablePageType.name);
+  await pageTypesBuilderPage.goto(singlePageType.name);
 
   await expect(
-    pageTypesBuilderPage.getFieldMenuButton("uid"),
+    pageTypesBuilderPage.getListItemFieldId("uid"),
   ).not.toBeVisible();
 });
 
-test("I cannot see default UID field for single page type", async ({
+test("I can edit the UID field", async ({
```

Comment:
Was considering that, will add!

Reply:
Did you add it? I don't see the check to verify that the UID edit button is not here on a single page type?
I only see the check that the field is not visible in the static fields.

Reply:
Gosh, I misunderstood your comment.
Will add that
anyway, will need to push fixes to e2e tests - two are failing on github :( which is strange because locally they work, need to investigate that